example/package.json

```diff
@@ -3,8 +3,7 @@
   "description": "Gatsby default starter",
   "version": "1.0.0",
   "dependencies": {
-    "gatsby": "^2.32.12",
-    "gatsby-plugin-algolia": "link:..",
+    "gatsby": "^4",
```

Comment:
Is it just to update the example or do we only support >4 now?

Reply:
just the example, Gatsby v3 and v2 have the same cache api

Reply:
Ok I was just wondering